RESULTS.md

```diff
@@ -48,14 +48,16 @@ GPU: Tesla V100
 | Cirq                 | 0.7.0   |
 | PennyLane            | 0.8.1   |
 | QuEST (pyquest-cffi) | 0.1.1   |
+| JKQ DDSIM            | master  |
```

Comment:
master is not a fixed version, thus I don't think we should use this in the version, if you don't have a tagged version, a git commit SHA would also do the job.

Reply:
I added a tag and referenced it in the table.